config/alphagov.yml

```diff
@@ -229,6 +213,29 @@ govuk-frontenders:
     - govuk_publishing_components
     - stylelint-config-gds
 
+govuk-interaction-and-personalisation:
+  channel: '#interaction-and-personalisation-govuk'
```

Comment:
Please change this to #tech-interaction-and-personalisation, I don't want out non-tech colleagues getting spammed with PRs

Reply:
Updated as requested :)